gmgc_mapper/output_visualization/index.ipynb

```diff
@@ -74,8 +130,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "gb['num_hits'] = pd.cut(x = gb['times_gene_hit'], bins = [0, 10, 25, 50, 100], \n",
```

Comment:
I am not sure I understand the bins here are up to 100, but then later the largest label is `>50 hits`. Is this just something that happened in your particular example?